src/System.Security.Permissions/src/System/Security/HostProtectionException.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace System.Security
+{
+    public partial class HostProtectionException : System.SystemException
+    {
+        public HostProtectionException() { }
+        // protected HostProtectionException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
+        public HostProtectionException(string message) { }
+        public HostProtectionException(string message, System.Exception e) { }
+        public HostProtectionException(string message, System.Security.Permissions.HostProtectionResource protectedResources, System.Security.Permissions.HostProtectionResource demandedResources) { }
+        public System.Security.Permissions.HostProtectionResource DemandedResources { get { return default(System.Security.Permissions.HostProtectionResource); } }
+        public System.Security.Permissions.HostProtectionResource ProtectedResources { get { return default(System.Security.Permissions.HostProtectionResource); } }
```

Comment:
I dug up the data, and the only useage in our corpus of property getters/setters on CAS types is on the Attributes, and on SecurityElement. Usual caveats about the limits of the corpus we have, but this suggests to me that we pass on such wiring, and spend our time on other API. We can come back if we need to.

---
In reply to: [70818095](https://github.com/dotnet/corefx/pull/10049#discussion_r70818095) [](ancestors = 70818095)